tests/Resources/XliffTest.php

```diff
@@ -13,8 +13,6 @@
 
 namespace Sonata\AdminBundle\Tests\Resources;
 
-use Sonata\CoreBundle\Test\XliffValidatorTestCase;
```

Comment:
You can't just remove the use if it's used two lines after.

Reply:
@wbloszyk You deprecated the XliffValidatorTestCase here https://github.com/sonata-project/SonataCoreBundle/commit/16e8cc28efc527f4814ea2e2898d37ad512130ea#diff-f00775389b61bfae085fa663caf4d96b.

Did you plan to move it somewhere else or should we add the methods here ?

Reply:
this was done for other bundles, I think we missed here. The TestCase was moved to every bundle using it

Reply:
This class should be copy here. I did it on some bundles.

Reply:
@VincentLanglet This class is in AdminBundle too. I just did not remove CoreBundle usage.